src/main/java/mate/academy/springbootintro/service/cartitem/impl/CartItemServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.springbootintro.service.cartitem.impl;
+
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springbootintro.dto.cartitem.CartItemDto;
+import mate.academy.springbootintro.dto.cartitem.CreateCartItemRequestDto;
+import mate.academy.springbootintro.dto.cartitem.UpdateCartItemRequestDto;
+import mate.academy.springbootintro.mapper.CartItemMapper;
+import mate.academy.springbootintro.model.Book;
+import mate.academy.springbootintro.model.CartItem;
+import mate.academy.springbootintro.model.ShoppingCart;
+import mate.academy.springbootintro.repository.book.BookRepository;
+import mate.academy.springbootintro.repository.cartitem.CartItemRepository;
+import mate.academy.springbootintro.repository.shoppingcart.ShoppingCartRepository;
+import mate.academy.springbootintro.service.cartitem.CartItemService;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class CartItemServiceImpl implements CartItemService {
```

Comment:
You don't need to implement this service CartItemServiceImpl
All actions related to Shop cart should be in Shop cart service